src/game.js

```diff
@@ -209,6 +218,14 @@
             // reset the frame counter
             frameCounter = 0;
             frameRate = ~~(0.5 + 60 / me.sys.fps);
+            
+            // set step size based on the updatesPerSecond
+            stepSize = (1000 / Math.min(me.sys.updatesPerSecond, me.sys.fps));
```

Comment:
Remember my comment https://github.com/melonjs/melonJS/issues/99#issuecomment-126951678?

That's how I coded it.

`me.sys.fps`  // Max fps (draw **and update**)